Sources/FioriARKit/ARCards/Models/CardItemModel.swift

```diff
@@ -28,3 +28,53 @@ public protocol ActionTextComponent {
 public protocol IconComponent {
     var icon_: Image? { get }
 }
+
+internal struct CodableCardItem: Codable {
+    var id: String
+    var title: String
+    var descriptionText: String?
+    var detailImage: Data?
+    var actionText: String?
+    var icon: String?
+}
+
+public struct DefaultCardItem: CardItemModel {
```

Comment:
Proposing `DecodableCardItem` as name

Reply:
I'm considering the usefulness of also having it encodable but focused on decoding the json first. For now I can't think of a reason the content will change for us to provide a new json as a convenience for the developer. I'm also unsure of how to encode the `Image` back to base64.

If this is not necessary then I think 'DecodableCardItem' is reasonable.

Reply:
Please use either `DecodableCardItem` or `CodableCardItem` as name but not `DefaultCardItem`

Reply:
Renamed to DecodableCardItem